resin/models/device.py

```diff
@@ -17,6 +18,8 @@ class Device(object):
 
     """
 
+    RESIN_CREATION_YEAR = 2013
```

Comment:
2013 used below is arbitrary, it doesn't have to be a constant. It could be 2000 and it'll still work. I suggest you inline 2013.